core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/ProjectImpl.java

```diff
@@ -64,6 +64,7 @@ public class ProjectImpl implements Project {
   private String title = "";
   private String code;
   private String rebNumber;
+  private String additionalDetails = "";
```

Comment:
Don't initialize to empty String. I don't know why some of the other fields do this, but there might be some old code that depends on it for those